lib/src/widgets/yaru_tab_bar.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:yaru_widgets/yaru_widgets.dart';
+
+/// A pre-styled replacement for material [TabBar]
+class YaruTabBar extends StatelessWidget {
+  const YaruTabBar({
+    super.key,
+    required this.tabController,
+    required this.tabs,
+    this.height = kYaruTitleBarItemHeight + 10,
```

Comment:
I think it would be better to allow a null value, and set the default height in a final var inside the `build` method.